pyproject.toml

```diff
@@ -82,7 +82,7 @@ dynamic = [
 ]
 dependencies = [
   "backports.zoneinfo<1; python_version<'3.9'",
-  "crate>=1.0.0.dev2,<2",
+  "crate>=2.0.0.dev1,<3",
```

Comment:
In order to permit installation of crate-2.0.0, we need to bump the upper version bound anyway. Do you think we should also bump the lower bound, as outlined here?

Reply:
For me it sounds very fine to increase the lower bound, everything else looks like not worth doing it (I don't really see the need to be able to use an older crate-python version with a newer sqlalchemy dialect)

Reply:
Fine. Let's merge the PR as-is then?